easybuild/easyconfigs/i/Iris/Iris-1.11.0-intel-2016a-Python-3.5.1.eb

```diff
@@ -0,0 +1,43 @@
+easyblock = 'PythonPackage'
+
+name = 'Iris'
+version = '1.11.0'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'http://scitools.org.uk/iris/'
+description = """Iris seeks to provide a powerful, easy to use, and community-driven Python library for analysing and visualising meteorological and oceanographic data sets."""
+
+toolchain = {'name': 'intel', 'version': '2016a'}
+
+source_urls = ['https://github.com/SciTools/Iris/archive/']
```

Comment:
seems like this one is not available on PyPI, so OK (the Iris that *is* there is something else, I think)